x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/empty_prompt/empty_prompt.tsx

```diff
@@ -63,21 +63,34 @@ export const AnalyticsEmptyPrompt: FC<Props> = ({ disabled, onCreateFirstJobClic
           <p>
             <FormattedMessage
               id="xpack.ml.overview.analyticsList.emptyPromptText"
-              defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis and put the annotated data in a new index. The classification and regression trained models can also be used for inference in pipelines and aggregations."
+              defaultMessage="Build and train outlier detection, regression, or classification machine learning models using data frame analytics."
```

Comment:
Not mandatory, but IMO there's not a significant difference between "build" and "train" here, so I'd drop one:
```suggestion
              defaultMessage="Train outlier detection, regression, or classification machine learning models using data frame analytics."
```